src/mixins/labels.js

```diff
@@ -95,6 +96,72 @@ export const LabelsMixin = (superclass) =>
             });
         }
 
+        /*
+         * removes labels that situated in the labels array
+         */
+        removeLabels() {
+            this.#labels.forEach((label) => this.structureGroup.remove(label));
+        }
+
+        /**
+         * this function is creates vertices hashMap where key is atom name, value is array of vertices where this atoms is situated
```

Comment:
is creates -> creates
Why do we need this function? Let's explain in the docstring as well.